packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -3,83 +3,79 @@ id: Testing your accessibility
+ The [WAVE browser extension from WebAIM](//wave.webaim.org/extension/) provides this feature if it isn't available in the browser you are using.
 
-- **Links**: Similar to buttons, links should always have a label for clarity. Without the label, users have no idea where the links point to. If you have the same label for multiple links, each link must point to the same URL. Links, buttons, and form controls should make sense out of context. If a user wants to look at all of the links available, they should be able to differentiate the available links. You shouldn’t have multiple “Click here” links.
+### 5. Test with a screen reader 
 
-- **Landmarks**: Landmarks help communicate the structure of a page by identifying regions. There are many types of landmarks, such as banners, navigation, main, and form. When there are more than one of the same role (for example, two navigation regions on the same page), these regions should be differentiated by an `aria-label`.
+You can use any screen reader that is available in your operating system. To test PatternFly, we target the following screen readers:
+  - JAWS with Chrome, Windows ([JAWS keyboard shortcuts](//dequeuniversity.com/screenreaders/jaws-keyboard-shortcuts))
+  - Voiceover with Safari, Mac ([Voiceover keyboard shortcuts](//dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts))
+  - NVDA with Firefox, Windows ([NVDA keyboard shortcuts](//dequeuniversity.com/screenreaders/nvda-keyboard-shortcuts))
 
-- **Dynamic content**: One of the biggest accessibility concerns with dynamic content is the need to notify users that the content has changed. Sighted users benefit from highlighting or drawing visual attention to the changes. Non-sighted users need to be notified in another way, such as loading a new page, sending the focus to the new content, or using ARIA live announcements.
+### 6. Check color contrast
 
-- **Unrelated notifications**: Notifications from the operating system outside of the web app can interrupt a user while interacting with a page (for example, "You received a new chat message."). Consider this possibility when designing and developing for screen readers, and keep any notification messaging concise to limit the interruption.
+Colors should pass the following contrast checks, to ensure that users across the vision spectrum can understand your product:
+    - Text color against background color ([Understanding WCAG 1.4.3](//www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html))
+    - Text color against link color ([Technique G183](//www.w3.org/TR/WCAG20-TECHS/G183.html))
+    - Visible boundaries of buttons and form elements against adjacent background color ([Understanding WCAG 1.4.11](//www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html))
 
```

Comment:
I noticed that we removed the section on screen reader specific considerations. Do you think the information was redundant or not helpful? Perhaps we could take some of this information and put it underneath the screen reader section above to retain some of it? Curious of your thoughts on this one.

Reply:
Realizing as I continued reviewing that this was added to the development section. I have mixed feelings. I remember adding it so that users understood how things differed in a screen reader for testing, but I totally get that this also affects developing. @thatblindgeye what do you think? I'm worried that being closer to the content/writing it, I might not be able to see it as objectively. 😂

Reply:
Yeah that's tough because it can probably fit in either place. I feel like the content as it is *probably* fits closer in the development page. I guess it also depends if someone would be going through these pages in order or just what page interests them. I may have a clearer idea once I finish going through the current PRs but wondering whether linking to the development page would make sense? Sort of to convey "Also make sure you're testing stuff we mentioned on this page" or something, if that makes sense.

Reply:
Yeah, that would make sense to me! I think after reviewing them a second time, I agree that that screen reader content probably makes more sense in the development section.